session_0/word_processing.rb

```diff
@@ -18,7 +27,14 @@ def lower_case!(words)
 # words_with_prefix('apple', ['apple', 'ball', 'applesauce']) would
 # return the words 'apple' and 'applesauce'.
 def words_with_prefix(prefix, words)
-  raise NotImplementedError # TODO
+  #raise NotImplementedError # TODO
+  same = []
+  words.each do |i|
```

Comment:
Can you use select function here? https://www.rubyguides.com/2019/04/ruby-select-method/